--- origsrc/pytraffic-2.5.4/setup.py	2006-01-27 01:59:07.000000000 -0600
+++ src/pytraffic-2.5.4/setup.py	2007-03-11 20:10:07.812500000 -0500
@@ -58,7 +58,7 @@
         def build_and_install_pytraffic_script(self):
             main_command_contents=\
 r"""#!/bin/sh
-exec python %s/share/%s/Main.py""" %\
+exec python %s/lib/%s/Main.py""" %\
                    (os.path.abspath(self.install_base),config_db["install_leaf"])
             pytraffic_build=os.path.join(self.build_base,"pytraffic")
             pytraffic_install=os.path.join(self.install_base,
@@ -150,8 +150,8 @@
                         'install_lib' :  "c:\\"+config_db["install_leaf_windows"]}
     options['build_ext']={'compiler' : 'mingw32'}
 else:
-    options['install']={'prefix' : '/usr/local',
-                        'install_lib' : '$base/share/'+config_db["install_leaf"],
+    options['install']={'prefix' : '/usr',
+                        'install_lib' : '$base/lib/'+config_db["install_leaf"],
                        'install_data' : '$base'}
     options['build_ext']={'compiler' : 'unix'}
 
